pom.xml

```diff
@@ -91,6 +91,89 @@
         <gem-maven-plugin.version>1.0.5</gem-maven-plugin.version>
```

Comment:
❓ Should we add a plugin properties section and move the other down below to their corresponding sections?

Reply:
Will add